src/components/DataItemList/DataItemsList.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import DataItem, { DataItemsProps } from "../DataItem/DataItem";
+
+export type DataItemsListProps = {
+  itemsList: DataItemsProps[];
+  className?: string;
+}
+
+export const DataItemsList = React.forwardRef<HTMLDivElement, DataItemsListProps>(function DataItemList(
```

Comment:
DataItems might be better. WDYT?

Reply:
basically it's a list, but can be just items

Reply:
I like DataItemsList, but however you guys want 😂